setup.py

```diff
@@ -59,7 +59,7 @@ def find_files(directory):
           'pyqtgraph',
```

Comment:
This is the closest place to add the comment. On line 56 is the EXtra-geom dependency, should this be set to >= version 1.1.0 (or whatever version it is that had the breaking changes?)

Reply:
Thanks, good point. I think 1.0 may have broken it, but the `.offset()` method is new in 1.1, so that's the floor.